Source/Core/HeightmapTerrainData.js

```diff
@@ -182,7 +182,7 @@ Object.defineProperties(HeightmapTerrainData.prototype, {
   },
 });
 
-var taskProcessor = new TaskProcessor("createVerticesFromHeightmap");
+var taskProcessor = new TaskProcessor("createVerticesFromHeightmap", 5000);
```

Comment:
Ahhh this previous limit of 5 was interfering with tests or the sand castle or something... I can't really remember 😛  I guess because every tile now has to go across the worker (during `createMesh`), there's a lot more traffic, and therefore chances for that limit of 5 to be reached pretty quick. Will revert / figure it out later.